bin/libs/certificate.sh

```diff
@@ -199,6 +199,8 @@ pkcs12_create_certificate_authority() {
   password="${3}"
   common_name=${4:-${ZWE_PRIVATE_DEFAULT_CERTIFICATE_CA_COMMON_NAME}}
 
+  flags=$(get_java_pkcs12_keystore_flag)
+
   print_message ">>>> Generate PKCS12 format local CA with alias ${alias}:"
   mkdir -p "${keystore_dir}/${alias}"
   result=$(pkeytool -genkeypair -v \
```

Comment:
Do you think that this flag should be also specified in functions that cover exports from keyring? like keyring_export_to_pkcs12

Reply:
i was not sure if it was allowed, but probably it is. i can try it.